sites/all/themes/wille/template.php

```diff
@@ -112,18 +112,22 @@ function wille_ting_collection_view_alter(&$build) {
  * Wraps the cover in a link to the material.
  */
 function wille_ting_object_cover($variables) {
+  $ding_entity_id = $variables['elements']['#object']->ding_entity_id;
```

Comment:
Why have you moved `$ding_entity_id` to the top? You don't need it in your new code, so it should be left in its original place.